packages/frontpage/app/(app)/layout.tsx

```diff
@@ -133,7 +133,6 @@ async function LoginOrLogout() {
               await signOut();
               deleteAuthCookie(cookies());
               revalidatePath("/", "layout");
-              redirect("/");
```

Comment:
Removed this because there's a bug in next.js that doesn't revalidate layouts when combined with a redirect.

https://github.com/vercel/next.js/issues/70483